lib/oppen/token.rb

```diff
@@ -21,6 +21,8 @@ module BreakType
     # Default token width
     # @return [Integer]
     def width = 0
+    # @return [Boolean] Whether the token can be trailing.
+    def trailing? = true
```

Comment:
`trailing` only means that it's something _"hanging loosely"_.

The way you're using it elsewhere makes it confusing: a whitespace is not trailing, but a non-whitespace is?

I don't exactly know how to replace that term, we may need to have a call to iron this one out.